server/src/modules/post/post.service.ts

```diff
@@ -41,17 +41,44 @@ export class PostService {
     return ['views', 'DESC']
   }
 
+  private getPaginatedPosts = (
+    post: Post[],
```

Comment:
nitpick
```suggestion
    posts: Post[],
```
and elsewhere